public/humans.txt

```diff
@@ -1,2 +1,13 @@
 GOV.UK is built by a team at the Government Digital Service (with thanks to Martha!) in London,
 Bristol and Manchester.  If you'd like to join us, see https://gdscareers.gov.uk/
+
+Today we're saying goodbye to a senior product manager, Mark McLeod. Mark joined GDS in 2012
+as a policy advisor, and later became a product manager in the early stages of GOV.UK.
+
+Mark has worked on many things, including search, email notifications and retiring Local
+Directgov. He's also been instrumental in giving GOV.UK publishers access to data about their
```

Comment:
I think you can make it slightly snappier by removing `also` :)

```suggestion
Directgov. He's been instrumental in giving GOV.UK publishers access to data about their
```

Reply:
Sure, updated 👍.